db/models.py

```diff
@@ -1 +1,34 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="skill_set")
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="players")
+    guild = models.ForeignKey(Guild,
+                              on_delete=models.SET_NULL,
+                              null=True,
+                              blank=True,
```

Comment:
do you need blank here?

Reply:
It may not be of practical use, but when creating a player, the guild field will be empty in the form, and after creation it will be null until the player joins the guild. But in practice, it can be taken away to perform the task.